### PR TITLE
Portfolio Desktop Version.

### PR DESCRIPTION
**Pull request by kidde60**
### **Portfolio Desktop-Version Webpage.**
**On this webpage I:**

- added a README.md file.
- added sections.
- added a contact form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+</div>
+<div class="contact">
+    <ul>
+        <li><img src="image/Vector.png" alt="Github" class="media"></li>
+        <li><img src="image/Icon-Linkedin.png" alt="in" class="media"></li>
+        <li><img src="image/angelist-1.png" alt="angelist" class="media"></li>
+        <li><img src="image/Icon-Twitter.png" alt="Twitter" class="media"></li>
+        <li><img src="image/Icon-Medium.png" alt="M" class="media"></li>
+    </ul>
+</div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please add a link with an `a tag` to all your social apps.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please follow the figmal [Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) to match your recent work section.
### **Take A Look**
urs
### **Figmal Design**
rest

File: index.html [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it. 
+    </p>
+    <form  action="https://formspree.io/f/mnqwoyza" method="POST">
+        <input id="name" name="Name" class="text" type="text" placeholder="Full Name" maxlength="30" required>
+        <input id="email" name="email" class="text" type="email"  placeholder="example@gmail.com" required>
+        <textarea name="Message" id="message" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+        <button type="button" id="btn" class="button">See Project</button>
+    </form>
+</section>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please follow the figmal [Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) to match your contact form section.
### **Take A Look**
form
### **Figmal Design**
fig=form

File: style.css [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+  .content {
+    vertical-align: top;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: url("images/circle1.svg");
+    background-repeat: no-repeat;
+    background-position: left;
+  }
+
```
Comment (CollinsTatang) [inline style.css]: - [ ] Kindly fix the responsive nature of your app to fit different screen sizes as expected.
### **See Mobile View**
mob

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Kidde60, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 


- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kidde60** [comment]
We were advised to use only one input for the full name.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+            </ul>            
+        </nav>
+    </header>
+<section class="content">
+        <h1>Kidde George</h1>
+        <h2>Software Developer</h2>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+</section>
+<div class="img">
+    <img src="images/Vector4.png" alt="line">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+         <li>Github</li>
+         <li>Codepen</li>
+         <li>Gitlab</li>
+         <li>Terminal</li>
+     </ul>
+    </div>
+
+</section>
+<div class="img">
+    <img src="images/Vector4.png" alt="line" class="img" id="img1">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+        <img src="image/arrow.svg" alt="down-arrow"/> 
+    </div>
+    <div class="bar, bar1">
+        <img src="image/arrow.svg" alt="down-arrow"/>
+        <h5>Work</h5> 
+    </div>
+</div>
+<div>
+    <h2 class="work">MY RECENT WORK</h2>
+    <img src="images/Vector5.png" alt="bar" class="img" id="line">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+    </p>
+    <form  action="https://formspree.io/f/mnqwoyza" method="POST">
+        <input id="name" name="Name" class="text" type="text" placeholder="Full Name" maxlength="30" required>
+        <input id="email" name="email" class="text" type="email"  placeholder="example@gmail.com" required>
+        <textarea name="Message" id="message" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+        <button type="button" id="btn" class="button">See Project</button>
+    </form>
+</section>
+<div class="icon">
+    <img src="images/Vector5.png" alt="line" class="img" id="icon">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+    </div>
+</div>
+<div>
+    <h2 class="work">MY RECENT WORK</h2>
+    <img src="images/Vector5.png" alt="bar" class="img" id="line">
+</div>
+<section class="work-section">
+    <div class="placeholder">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
```
Comment (ShahierNashaat) [inline index.html:91]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+    
+    <div class="background-image1">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
```
Comment (ShahierNashaat) [inline index.html:109]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,193 @@
...
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image2">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
```
Comment (ShahierNashaat) [inline index.html:127]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,193 @@
...
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image3">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
```
Comment (ShahierNashaat) [inline index.html:145]: - [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @kidde60 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @kidde60,

Good job on working on all the changes asked by the previous reviewer.

 YesYouAreGIF

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**